test/hotspot/jtreg/compiler/intrinsics/sha/sanity/DigestSanityTestBase.java

```diff
@@ -54,7 +54,7 @@ public class DigestSanityTestBase {
     private static final int MSG_SIZE = 1024;
     private static final int OFFSET = 0;
     private static final int ITERATIONS = 10000;
-    private static final int WARMUP_ITERATIONS = 550;
+    private static final int WARMUP_ITERATIONS = WHITE_BOX.getIntxVMFlag("Tier4InvocationThreshold").intValue() + 50;
```

Comment:
If the flag `Tier4InvocationThreshold` is not set, what will happen?

Reply:
The default value, 5000, would be returned.